pkg/apis/pipeline/v1beta1/pipeline_types_test.go

```diff
@@ -279,6 +279,7 @@ func TestPipelineTask_ValidateRegularTask_Success(t *testing.T) {
 				cfg.FeatureFlags.EnableTektonOCIBundles = true
 			}
 			ctx = config.ToContext(ctx, cfg)
+			ctx = context.WithValue(ctx, ValidateParameterVariablesAndWorkspaces, true)
```

Comment:
I'm curious why this validation is going in a call to `context.withvalue` (as opposed to being part of (PipelineSpec).Validate), and why it is added to tests?